README.md

```diff
@@ -5,1560 +5,16 @@
-
-```
-[{
-    "profiles": [
-        {
-            "name": "100% packet loss",
-            "identifier": "SlowNetwork100PctLoss",   // enableConditionInducer profileID
-            "description": "Name: 100% Loss Scenario
-                            Downlink Bandwidth: 0 Mbps
-                            Downlink Latency: 0 ms
-                            Downlink Packet Loss Ratio: 100%
-                            Uplink Bandwidth: 0 Mbps
-                            Uplink Latency: 0 ms
-                            Uplink Packet Loss Ratio: 100%"
-        },
-    ],
-    "profilesSorted": true,
-    "identifier": "SlowNetworkCondition",   // enableConditionInducer conditionID
-    "isDestructive": false,
-    "isInternal": false,
-    "activeProfile": "",
-    "name": "Network Link",
-    "isActive": false
-}]
-```
-
-### mobile: disableConditionInducer
-
-Disable device condition inducer.
-
-Usually a persistent connection is maintained after enable the condition inducer, and this method is only valid for this connection.
-
-If the connection is disconnected, condition inducer will be automatically disabled
-
-#### Returned Result
-
-Either `true` or `false`, where `true` means disabling of the condition inducer has been successful
-
-### mobile: updateSafariPreferences
-
-Updates preferences of Mobile Safari on Simulator
-
-#### Arguments
-
-Name | Type | Required | Description | Example
---- | --- | --- | --- | ---
-preferences | map | yes | An object containing Mobile Safari preferences to be updated. The list of available setting names and their values could be retrieved by changing the corresponding Safari settings under Preferences-&gt;Safari and then inspecting `Library/Preferences/com.apple.mobilesafari.plist` file inside of `com.apple.mobilesafari` app container. The full path to the Mobile Safari's container could be retrieved from `xcrun simctl get_app_container <sim_udid> com.apple.mobilesafari data` command output. Use the `xcrun simctl spawn <sim_udid> defaults read <path_to_plist>` command to print the actual .plist content to the Terminal. | { ShowTabBar: 0, WarnAboutFraudulentWebsites: 0 }
-
-## Known issues
-
-* `shake` is implemented via AppleScript and works only on Simulator due to lack of support from Apple
-
-
-## Sim Resetting
-
-By default, this driver will create a new iOS simulator and run tests on it, deleting the simulator afterward.
-
-If you specify a specific simulator using the `udid` capability, this driver will boot the specified simulator and shut it down afterwards.
-
-If a udid is provided and the simulator is already running, this driver will leave it running after the test run.
-
-In short, this driver tries to leave things as it found them.
-
-You can use the `noReset` capability to adjust this behavior.
-Setting `noReset` to `true` will leave the simulator running at the end of a test session.
 
+Next, read the [Full Documentation](https://appium.github.io/appium-xcuitest-driver)!
```

Comment:
Can we reflect the full TOC in the readme?

Reply:
we could, although it would probably need to be maintained by hand and would likely get out of sync if we're not careful. but automatically generating it from the mkdocs metadata would be a decent amount of engineering and i'm not sure it's worth it.

Reply:
I assume it might make it easier for users to find what they are looking for...

Reply:
Yes, I agree, and if there were an easy way to autogenerate this I would say it's a good idea. But for now clicking through one link before seeing the TOC is probably not an awful experience. it's what a lot of READMEs do for projects that have independent docs sites.